Dockerfile

```diff
@@ -1,20 +1,23 @@
 FROM ubuntu:22.04
 
 # Constants
-ARG VERSION_RUST="nightly-2022-12-08"
+# TODO: change this back after tests
+ARG VERSION_RUST="nightly-2022-10-16"
```

Comment:
Yes, for `v4.1.0` of the reproducible builds setup, we'll use Rust `2022-10-16` (used by most deployed contracts, as far as we know). Immediately after publishing `v4.1.0`, we'll also publish `v4.2.0` that has Rust `2022-12-08`. And from now on, we'll bump the minor version when important dependencies change (e.g. the Rust compiler). The "patch" segment of the version will be reserved for actual patches to the deterministic builds flow.